proto/umee/uibc/v1/events.proto

```diff
@@ -2,16 +2,20 @@ syntax = "proto3";
 package umee.uibc.v1;
 
 import "gogoproto/gogo.proto";
+import "umee/uibc/v1/quota.proto";
 
 option go_package                      = "github.com/umee-network/umee/v4/x/uibc";
 option (gogoproto.goproto_getters_all) = false;
 
 // EventBadRevert is emitted on failure of ibc-transfer quota.
 message EventBadRevert {
-  // module name
-  string module = 1;
   // failure event type
   string failure_type = 2;
   // ibc packet data
   string packet = 3;
```

Comment:
```suggestion
  // failure event type
  string failure_type = 1;
  // ibc packet data
  string packet = 2;
```
Since we're removing the field above anyway, and this isn't on mainnet, we can fix the numbers without worrying about breakage.

Reply:
makes sense